File: LogicaDeProgramacaoEclipse(Java)/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/exerciciosFaccat/Exercicio31.java
package exerciciosFaccat;

import java.util.Scanner;

/*31) Ler 3 valores (A, B e C) representando as medidas dos lados de um triângulo e escrever se formam
// ou não um triângulo. OBS: para formar um triângulo, o valor de cada lado deve ser menor que a soma
// dos outros 2 lados. */

public class Exercicio31 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		Float soma1;
		Float soma2;
		Float soma3;
		Float valorA;
		Float valorB;
		Float valorC;

		System.out.println("Digite o valor A");
		valorA = sc.nextFloat();
		System.out.println("Digite o valor B");
		valorB = sc.nextFloat();
		System.out.println("Digite o valor C");
		valorC = sc.nextFloat();

		soma1 = valorB + valorC;
		soma2 = valorA + valorC;
		soma3 = valorA + valorB;

		if ((valorA < soma1) && (valorB < soma2) && (valorC < soma3)) {
			System.out.println("essas medidas formam um tringulo");
		} else {
			System.out.println("essas medidas não formam um tringulo");
		}

		sc.close();
	}

}

File: LogicaDeProgramacao(JavaScript)/Exercicios 5 até o 39 Apostila Faccat/js/exercicio31.js
/*31) Ler 3 valores (A, B e C) representando as medidas dos lados de um triângulo e escrever se formam
// ou não um triângulo. OBS: para formar um triângulo, o valor de cada lado deve ser menor que a soma
// dos outros 2 lados. */

var soma1 = 0
var soma2 = 0
var soma3 = 0

var valorA = parseFloat(prompt("Digite o valor A"))
var valorB = parseFloat(prompt("Digite o valor B"))
var valorC = parseFloat(prompt("Digite o valor C"))


   soma1 = valorB + valorC
   soma2 = valorA + valorC
   soma3 = valorA + valorB

   if ((valorA < soma1) && (valorB < soma2) && (valorC < soma3)) {
      alert("essas medidas formam um tringulo")
   } else {
      alert("essas medidas não formam um tringulo")
   }